dev/com.ibm.ws.jaxws.2.2.webcontainer_fat_extended/test-applications/encodingTest/src/com/ibm/ws/jaxws/test/wsr/test/servlet/EncodingHttpTestServlet.java

```diff
@@ -0,0 +1,145 @@
+/*******************************************************************************
+ * Copyright (c) 2024 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package com.ibm.ws.jaxws.test.wsr.test.servlet;
+
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import javax.annotation.Resource;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.xml.ws.BindingProvider;
+import javax.xml.ws.handler.MessageContext;
+
+import com.ibm.ws.jaxws.test.wsr.server.stub.People;
+import com.ibm.ws.jaxws.test.wsr.server.stub.PeopleService;
+
+/**
+ *
+ */
+@WebServlet("/EncodingHttpTestServlet")
+public class EncodingHttpTestServlet extends HttpServlet {
```

Comment:
To following a convetion we've been using, could we rename the servlet to just be `junit test class name + servlet`?

```suggestion
public class EncodingTestServlet extends HttpServlet {
```